doc/sphinx/proof-engine/ltac.rst

```diff
@@ -1151,34 +1639,51 @@ Tactic toplevel definitions
 Defining |Ltac| functions
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Basically, |Ltac| toplevel definitions are made as follows:
+|Ltac| toplevel definitions are made as follows:
 
-.. cmd:: {? Local} Ltac @ident {* @ident} := @ltac_expr
+.. cmd:: Ltac @tacdef_body {* with @tacdef_body }
    :name: Ltac
 
-   This defines a new |Ltac| function that can be used in any tactic
+   .. insertprodn tacdef_body tacdef_body
+
+   .. prodn::
+      tacdef_body ::= @qualid {* @name } {| := | ::= } @ltac_expr
+
+   Defines a new |Ltac| function that can be used in any tactic
```

Comment:
```suggestion
   Defines or redefines an |Ltac| function that can be used in any tactic
```

Reply:
Nope, doesn't permit redefinition.  The message is documented a few lines below.